build_tools/buildkite/cmake/linux/pipeline.yml

```
@@ -109,15 +109,19 @@ steps:
   - label: ":lower_left_crayon: Comment benchmark results on pull request"
     key: "post-on-pr"
     commands: |
-      git clean -fdx
-      buildkite-agent artifact download "benchmark-results-*.json" ./
-      buildkite-agent artifact download "compile-stats-results-*.json" ./
+      set -euo pipefail
+      # Try to download the artifacts and not fail if not found.
```

nit: tweak phrasing? ("and not fail if not found" is a bit strange to read)

```suggestion
      # Try to download the artifacts, continuing with a message if not found.
```

---

Looks good. Thx!